CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/GraphConstructionDemoAction.java

```diff
@@ -94,7 +94,7 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
 
                     final int[] childrenAtDepths = {4, 3, 2};
                     final TreeGraphBuilder t = TreeGraphBuilder.addTree(graph, childrenAtDepths, TreeGraphBuilder.TreeDirection.AWAY_FROM_ROOT);
-                    final ConnectionBuilder con = ConnectionBuilder.makeConnection(graph, t.nodesAtLevels[0], j.pendants);
+                    ConnectionBuilder.makeConnection(graph, t.nodesAtLevels[0], j.pendants);
```

Comment:
Builds a connection between the two nodes on the graph but its unnecessary to store the variable itself.